Find a functionally equivalent code:android.content.RestrictionEntry.getKey:COMMENT
Method Modifier: public      
Comment:/**
 * This is the unique key for the restriction entry.
 * @return the key for the restriction.
 */

@@ -1,3 +1,3 @@
 {
-    return key;
+    return mKey;
 }

